src/chart/candlestick/CandlestickView.ts

```
@@ -110,6 +110,9 @@ class CandlestickView extends ChartView {
             .add(function (newIdx) {
                 if (data.hasValue(newIdx)) {
                     const itemLayout = data.getItemLayout(newIdx) as CandlestickItemLayout;
+                    if (!itemLayout) {
```

If everything goes well, `itemLayout` should never be null. If we simply ignore it, potential errors might be hidden unexpectedly. I suggest not adding this non-null check.

---

Sounds good, will remove those checks.